src/styles/blocks/clock.scss

```
@@ -0,0 +1,95 @@
+@use "../utils/extends";
+@use "../utils/variables" as *;
+
+body {
+  margin: 0;
+  padding: 0;
+  height: 100vh;
+  background-color: #FFEEDB;
+  @extend %centre-flex;
+  font-size: 44px;
+  font-weight: 500;
+  font-family: "Roboto", sans-serif;
+}
+
+.watch {
+  position: relative;
+  @extend %watch-size;
+  @extend %watch-radius;
+  background-color: #F9DC5C;
+  border: 16px solid #0b4f6c43;
+}
+
+.watch__centre {
+  @extend %centre-flex;
+  @extend %watch-centre;
+  flex-direction: column;
+  text-transform: uppercase;
+  @extend %watch-radius;
+  height: calc(#{$watch-size * 0.095});
```

- you don't need `#{}` wrapper in SCSS
- create a variable out of 0.095
```suggestion
  $scale-primary: 0.095;

  height: calc($watch-size * $scale-primary);
```